src/git-town

```diff
@@ -31,7 +31,7 @@ function show_help {
 
 
 function show_version {
-  echo "Git Town 0.5.0"
+  echo "Git Town 0.5.0$(gittown_dev_version)"
```

Comment:
This looks like dev_version is always printed when thats not the case. How about
```sh
local version="0.5.0"
if [ "$(is_manual_install)" == "true" ]; then
  version="$version ($revision_info)"
fi
echo "Git Town $version"
```